src/status_im/ui/screens/appearance/views.cljs

```diff
@@ -11,7 +11,7 @@
   [react/touchable-highlight
    {:on-press (fn []
                 (re-frame/dispatch [:multiaccounts.ui/appearance-switched theme])
-                (re-frame/dispatch [:init-root :chat-stack])
```

Comment:
i feel like we need to move `(if @config/new-ui-enabled?` here, and keep current code for old ui?

Reply:
Sure, I've tested and it works, but it is probably safer to keep the code for the old UI. Will make the change.

Reply:
Done